src/main/java/graphql/execution/instrumentation/parameters/InstrumentationExecutionParameters.java

```diff
@@ -26,16 +27,32 @@ public InstrumentationExecutionParameters(ExecutionInput executionInput, Instrum
                 executionInput.getContext(),
                 executionInput.getVariables() != null ? executionInput.getVariables() : Collections.emptyMap(),
                 instrumentationState);
+        this.executionInput = executionInput;
     }
 
-    public InstrumentationExecutionParameters(String query, String operation, Object context, Map<String, Object> variables, InstrumentationState instrumentationState) {
+    private InstrumentationExecutionParameters(String query, String operation, Object context, Map<String, Object> variables, InstrumentationState instrumentationState) {
```

Comment:
This constructor seems only ever used in the constructor above... so it should probably be removed?